src/TableView.elm

```diff
@@ -368,6 +381,73 @@ viewCells dateWithSessions model timeDelimiters numColumns timeDelimiter =
         |> List.indexedMap (viewCell dateWithSessions model timeDelimiters numColumns timeDelimiter)
 
 
+add0ToSingleDigits number =
```

Comment:
I think this function may already exist

Reply:
in the main application with have a v.similar function